lib/fizzbuzz.rb

```diff
@@ -1,9 +1,15 @@
 class SuperFizzBuzz
 
   def run(input)
-
-    #Implement your code here
-
+    if (input % 3 == 0) && (input % 5 == 0) 
+      "FizzBuzz"
+  	elsif input % 5 == 0
+  	  "Buzz"
+    elsif input % 3 == 0
+  	  "Fizz"
+  	else
+  	  input
+  	end 
   end
```

Comment:
Gotcha. It's not a *huge* deal, but improperly formatted code tends to raise a developer's urge to kill. :stuck_out_tongue_closed_eyes: 

Also, have you checked out the Atom editor? It's what a lot of the cool kids are using now, and it's the listed editor on the Getting Started page.

http://paircolumbus.org/getting_started/